wallet/src/ui/app/pages/home/receipt/index.tsx

```diff
@@ -20,23 +21,29 @@ import st from './ReceiptPage.module.scss';
 // use txDigest for the transaction result
 function ReceiptPage() {
     const [searchParams] = useSearchParams();
-
+    const dispatch = useAppDispatch();
     // get tx results from url params
     const txDigest = searchParams.get('txdigest');
 
-    const tranferType = searchParams.get('transfer');
+    const tranferType = searchParams.get('transfer') as 'nft' | 'coin';
```

Comment:
This (`as 'nft' | 'coin'`) doesn't look very safe as it can be anything, the user can change the url.

Reply:
Right now the transfer type is just used to display message related to either NFT or Coin.